aws/src/main/java/org/apache/iceberg/aws/glue/GlueCatalog.java

```diff
@@ -162,7 +167,21 @@ private String cleanWarehousePath(String path) {
 
   @Override
   protected TableOperations newTableOps(TableIdentifier tableIdentifier) {
-    return new GlueTableOperations(glue, lockManager, catalogName, awsProperties, fileIO, tableIdentifier);
+    if (catalogProperties != null) {
+      S3FileIO s3FileIO = new S3FileIO();
+      Map<String, String> lakeFormationProperties = ImmutableMap.<String, String>builder()
```

Comment:
this variable name can be more generic than just LF, something like `tableSpecificCatalogProperties`